src/logging/logging.ts

```diff
@@ -43,20 +35,45 @@ export class Log {
   constructor(private tag_:string) {
     this.logger = freedom.core().getLogger(this.tag_);
   }
+
+  private log_ = (level :string, args :any[]) :void => {
+    var message :string;
```

Comment:
It's often a good idea to declare variables closer to where they're actually used. In this case you can save a few lines and have the compiler infer the type for you.